tests/operators/test_agnostic_load_file.py

```diff
@@ -466,47 +468,94 @@ def test_aql_gcs_file_to_postgres(self):
         )
         assert df.iloc[0].to_dict()["sell"] == 142.0
 
-    def test_aql_local_file_to_snowflake(self):
+
+@pytest.fixture
+def sample_dag():
+    yield DAG("test_dag", default_args={"owner": "airflow", "start_date": DEFAULT_DATE})
```

Comment:
Huh... today I learned you can yield in a function and then keep going. Is there a benefit to doing it this way?

Reply:
My understanding is that this is the Pytest native way of doing fixtures (setup and teardown):
https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended
The first time I saw this, I thought it was strange. I guess I have gotten used to it since then - which may be good or not :joy: 

I believe the benefit, when compared to using `unittest.TestCase` traditional `setUp` and `tearDown` is only being able to use other Pytest features, as described in:
https://docs.pytest.org/en/6.2.x/unittest.html#pytest-features-in-unittest-testcase-subclasses